.pre-commit-config.yaml

```diff
@@ -19,15 +19,16 @@ repos:
   - id: lint-frontend
     name: Linting of JS, Vue, SCSS and CSS files on the frontend
     description: This hook handles all frontend linting for Kolibri
-    entry: yarn run lint-frontend
+    entry: yarn run lint-frontend:format
     language: system
     files: ^kolibri_explore_plugin/assets/src/.*\.(js|vue|scss|css)$
   - id: lint-packages
-    name: Linting of JS, Vue, SCSS and CSS files on the packages
+    name: Linting of JS and Vue files on the packages
     description: This hook handles all pacakges linting
-    entry: yarn run lint-frontend
+    entry: yarn run lerna run lint
```

Comment:
In my maintenance PR I have `yarn lint-packages` that does the same: https://github.com/endlessm/kolibri-explore-plugin/pull/74/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R11